index.js

```diff
@@ -208,19 +208,19 @@ export default class WebTorrent extends EventEmitter {
    * found.
    *
    * @param  {string|Buffer|Object|Torrent} torrentId
-   * @return {Torrent|null}
+   * @return {Promise<Torrent|null>}
    */
-  get (torrentId) {
+  async get (torrentId) {
     if (torrentId instanceof Torrent) {
       if (this.torrents.includes(torrentId)) return torrentId
     } else {
+      const torrents = this.torrents
```

Comment:
why

Reply:
since parsetorrent is now async, if someone called this function without waiting for async [happens frequently during tests] the torrent it was looking for might not longer exist, which would throw an error, here it just silent fails and returns null instead, TLDR did this so people aren't forced to use the await for this fn

Reply:
Ok, needs to be cleaned up

Reply:
@DiegoRBaquero that means?

Reply:
Eventually only support `await`, as this IS an `async` function